### PR TITLE
Desertion Screens

### PR DESCRIPTION
**Pull request by sophiedrake**
https://triadmoj.atlassian.net/browse/DIV-198

Move desertion screens to statement of case folder, create new exit screens and update existing screens

### CODE REVIEW

File: app/routes/v2.js [hunk 1]
```diff
@@ -360,29 +360,62 @@ var router = express.Router();
...
+    else {
+      res.redirect('../../exit/desertion/time-period');
     }
   });
 
-  router.post('/reason/desertion/details', function (req, res) {
+  router.post('/statement-of-case/desertion/details', function (req, res) {
     var desertionDetails = req.body.desertionDetails;
-    req.session.facts.desertionDetails = desertionDetails;
+    console.log(desertionDetails);
```
Comment (lukaszdrynski) [inline app/routes/v2.js]: remove console.log

File: app/views/v2/exit/desertion/agree.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+{% block heading %}
+You cannot continue with your application
+{% endblock %}
+
+{% block exit_reason %}
+  <p class="text">
+  For desertion to have taken place your {{ husbandOrWife() }} must have left without your consent.
+  </p>
+  <p class="text">
+      You may want to consider choosing <a href="../reason-for-divorce">another fact</a>.
```
Comment (lukaszdrynski) [inline app/views/v2/exit/desertion/agree.html]: need to move up one more directory level

File: app/views/v2/exit/desertion/time-period.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{% block heading %}
+You cannot continue with your application
+{% endblock %}
+
+{% block exit_reason %}
+  <p class="text">
+    You must have been deserted for a period of more than 2 years to use this as a grounds for divorce.
+  </p>
+  <p class="text">
+      You may want to consider choosing <a href="../reason-for-divorce">another fact</a>.
```
Comment (lukaszdrynski) [inline app/views/v2/exit/desertion/time-period.html]: same here ../../reason-for-divorce

**Comment by sophiedrake** [comment]
updated

**Comment by lukaszdrynski** [comment]
👍

**Pull request merged**